mail_activity_reminder/README.rst

```diff
@@ -23,7 +23,7 @@ Mail Activity Reminder
     :target: https://runbot.odoo-community.org/runbot/205/12.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```

Comment:
You have edited this manually and this shouldn't be touched

Reply:
It makes no difference - the file will be regenerated anyhow

Reply:
Yes, but more noise for this simple change. Can you do it at once?

Reply:
okay okay, you win as always :P, let me revert that specific file 👍

Reply:
done